### PR TITLE
Homework week 1 SAEED

### PR DESCRIPTION
**Pull request by SaeedRafay**

### CODE REVIEW

File: Week1/meetup.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    room_name VARCHAR(50),
+    floor_number INT
+  )`;
+  const CREATE_TABLE_MEETING = `CREATE TABLE IF NOT EXISTS ${tblMeeting} (
+    meeting_no INT NOT NULL PRIMARY KEY,
+    meeting_title VARCHAR(255) NOT NULL,
+    starting_time DATETIME,
+    ending_time DATETIME,
+    room_no INT,
+    FOREIGN KEY (room_no) REFERENCES ${tblRoom}(room_no)
```
Comment (GinaSt) [inline Week1/meetup.js:41]: Nicely done!

**Review by GinaSt — APPROVED** [review]
Hi Saeed! 

Very nice work, readable code, good understanding of the queries!
I have left some comments on things you can try it out.
Keep up the good work!

File: Week1/world-queries.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    // What are the names of countries that have “land” in their names?
+    const qryLand = `SELECT Name FROM country WHERE Name LIKE "%land%"`;
+    const qryLandPromise = await Promise.all([execQuery(qryLand)]);
+    const qryLandCountries = qryLandPromise[0]
+      .map((country) => country.Name)
+      .join(", ");
+    console.log(`Countries with "land" in their names: ${qryLandCountries} \n`);
+
+    // What are the names of the cities with population in between 500,000 and 1 million?
+    const qryCityPopulation = `SELECT Name FROM city WHERE POPULATION > 500000 AND POPULATION < 1000000`;
```
[the remark this answers is not in the document]
Comment (SaeedRafay) [reply Week1/world-queries.js]: I learned something new. Thanks! 😃

File: Week1/meetup.js [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+async function meetupDb() {
+  const dbName = "meetup";
+  const tblInvitee = "Invitee";
+  const tblRoom = "Room";
+  const tblMeeting = "Meeting";
+
+  const DROP_DATABASE = `DROP DATABASE IF EXISTS ${dbName}`;
+  const CREATE_DATABASE = `CREATE DATABASE IF NOT EXISTS ${dbName}`;
+  const CREATE_TABLE_INVITEE = `CREATE TABLE IF NOT EXISTS ${tblInvitee} (
+    invitee_no INT NOT NULL PRIMARY KEY,
```
[the remark this answers is not in the document]
Comment (SaeedRafay) [reply Week1/meetup.js]: I have updated the code accordingly. Thanks.

File: Week1/meetup.js [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+    invitee_name VARCHAR(50),
+    invited_by VARCHAR(50)
```
[the remark this answers is not in the document]
Comment (SaeedRafay) [reply Week1/meetup.js:28]: Yes, I have now learned the logic of what you have mentioned here through the lectures on Normalization. Thanks!

File: Week1/meetup.js (hunk 1 above)
Comment (SaeedRafay) [reply Week1/meetup.js:41]: Thanks!

**Comment by SaeedRafay** [comment]
@GinaSt Thank you for reviewing the homework and providing feedback. 🙇

**Pull request closed without merging**